core/runtime.rs

```diff
@@ -416,9 +421,38 @@ impl JsRuntime {
     self.inspector.as_mut().unwrap()
   }
 
+  pub fn global_realm(&mut self) -> JsRealm {
+    JsRealm::new(self.global_context())
+  }
```

Comment:
Why is it not stored on `JsRuntime`? Seems strange to call into `JsRealm::new` every time you want to access it

Reply:
Done